### PR TITLE
Make license pie chart more stylish

### PR DESCRIPTION
**Pull request by leslielazzarino**
Signed-off-by: Leslie Lazzarino <leslie.lazzarino@tngtech.com>

### Summary of changes
- shades of blue instead of unrelated, high saturation colors
- moved to the pie chart, as in the app we want to use as much as possible a limited number of shades, to have a more coherent look
- removed unused colors from the palette

New
NewPieChart
Old
OldPieChart

### Context and reason for change
I think the previous color scheme did look unpolished.

### How can the changes be tested
Go to the  projectstatistics popup.

### CODE REVIEW

File: src/Frontend/shared-styles.ts [hunk 1]
```diff
@@ -14,31 +16,22 @@ export const OpossumColors = {
   middleBlueOnHover: 'hsl(220, 41%, 60%)',
   darkBlue: 'hsl(220, 41%, 41%)',
   darkBlueOnHover: 'hsl(220, 41%, 65%)',
-  white: 'hsl(0, 0%, 100%)',
-  whiteOnHover: 'hsl(220, 41%, 65%)',
+  disabledButtonGrey: 'hsla(0, 0%, 0%, 0.13)',
+  lightGrey: 'hsla(0, 0%, 0%, 0.16)',
```
Comment (leslielazzarino) [inline src/Frontend/shared-styles.ts:20]: I wonder is we really need both line 19 and 20.

Comment (benedikt-richter) [reply src/Frontend/shared-styles.ts:20]: would be nice to get rid of one of these two

**Review by benedikt-richter — CHANGES_REQUESTED** [review]
For me as a user it is very difficult to map the segments of the pie charts to the elements of the legend. Actually, I had to compare the ordering on both to identify or to hover. 
Usability (for me) certainly dropped.

**Comment by vasily-pozdnyakov** [comment]
For me it is quite difficult to distinguish between the colors for complete/incomplete atrributions.
image

**Comment by sebathomas** [comment]
Just looking at the screenshot, why keep the critical licenses in that yellow-orange color? It's a bit hard to read and doesn't provide any information if they're all the same color.

**Pull request closed without merging**